tests/trainer/test_trainer_cli.py

```diff
@@ -88,3 +88,22 @@ def _raise():
 
     with pytest.raises(_UnkArgError):
         parser.parse_args(cli_args)
+
+
+# todo: add also testing for "gpus"
+@pytest.mark.parametrize(['cli_args', 'expected'], [
```

Comment:
add a test for early stopping where it's enabled --early_stopping

Reply:
add one for auto batch size as well

Reply:
done ^^